js/mapbox-gl.js

```diff
@@ -14,6 +14,7 @@ mapboxgl.Popup = require('./ui/popup');
 mapboxgl.Marker = require('./ui/marker');
 
 mapboxgl.Style = require('./style/style');
+mapboxgl.Source = require('./source/source');
```

Comment:
Can we expose `mapboxgl.addSourceType` instead of the whole `Source` module? There are lots of bits & pieces in `Source` that we don't want to be public.